v1/trending.yaml

```diff
@@ -9,6 +9,16 @@ paths:
         Provides a list of articles that are trending right now (receiving the most edits)
 
         Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      parameters:
+        - name: period
+          description: |
+            Optional number in hours to only see pages that became trending in
```

Comment:
Slightly clearer, perhaps: *The period, expressed in hours, for which to obtain the list of trending articles. Default: 1 hour*